pkg/kopia/cli/internal/opts/common_opts.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	defaultLogLevel = "error"
+	defaultLogLevel = "debug"
```

Comment:
Is this a temporary change only for debugging the PR?

Does `defaultLogLevel` apply to:
  1. the logs sent to the console;
  2. the logs sent to files;
  3. both 1 & 2?